third_party/BUILD

```diff
@@ -605,34 +605,35 @@ filegroup(
     ],
 )
 
-# Part of the Java tools remote archive. Not embedded or used in Bazel.
-pkg_tar(
-    name = "zlib_pkg",
-    srcs = ["//third_party/zlib:embedded_tools"],
-    extension = "tar.gz",
-    mode = "755",
-    # Create a new pkg_tar to preserve the directory structure (zlib/).
-    package_dir = "zlib",
-)
-
-# Part of the Java tools remote archive. Not embedded or used in Bazel.
-pkg_tar(
-    name = "ijar_pkg",
+filegroup(
+    name = "java_tools_archive",
+    srcs = select({
+        "//src/conditions:windows": [
+            ":java_tools.zip",
+        ],
+        "//conditions:default": [
+            ":java_tools.tar.gz",
+        ],
+    }),
+)
+
+genrule(
+    name = "java_tools_zip",
     srcs = [
-        "//third_party/ijar",
-        "//third_party/ijar:embed_into_java_tools",
-        "//third_party/ijar:zipper",
+        "//third_party/ijar:ijar_with_deps_zip",
+        "//third_party/java/java_tools:java_tools_zip",
+        ":zlib_zip",
     ],
-    extension = "tar.gz",
-    mode = "755",
-    # Create a new pkg_tar to preserve the directory structure (ijar/).
-    package_dir = "ijar",
+    outs = ["java_tools.zip"],
+    cmd = "$(location :merge_zip_files.sh) java_tools $(OUTS) $(SRCS)",
+    output_to_bindir = 1,
+    tools = [":merge_zip_files.sh"],
 )
 
 # Builds the remote Java tools archive. Not embedded or used in Bazel, but used
 # by the Java tools release process.
 pkg_tar(
-    name = "java_tools_pkg-gz",
+    name = "java_tools_tar",
```

Comment:
Is it possible to use zip on every platform instead of tar on some and zip on the rest?

Reply:
YES please.

Reply:
Upon further investigation I don't see any big advantage of keeping both the tar and the zip. The difference in size after compression is insignificant (42M vs 42.2M). The `pkg_tar` targets have been removed.

Reply:
/happydance